src/routes/papers/index.js

```diff
@@ -30,9 +30,8 @@ function getBadgeCount(request, response) {
     });
 
   getBadges(function (error, badges) {
-    if (error !== null) {
-      console.log('Get error from return Badges ' + error);
-      response.send(error);
+    if (error !== null || !badges) {
+      response.json(0);
```

Comment:
Why `0`? Are you trying to return nothing? If so do this instead `res.json(null)`